### DISCUSSION TITLE
TinyNvidiaUpdateChecker...Update Checker?

### DISCUSSION THREAD

**Discussion by nikandev** [ElPumpo/TinyNvidiaUpdateChecker#165] [Ideas]
Hey,
Can we get an update check step for the TNUC itself? 
Maybe using the [github release API](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release).

Thanks.

upd: My bad, just saw your open issue about it being in the works.

**Comment by HawaiiBeach** [comment]
TNUC already has a update checker for itself
